src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageDetails.tsx

```diff
@@ -3,64 +3,113 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { WorkPackage } from 'shared';
+import { WorkPackage, wbsPipe } from 'shared';
 import { percentPipe, fullNamePipe, datePipe, weeksPipe } from '../../../utils/pipes';
 import WbsStatus from '../../../components/WbsStatus';
-import PageBlock from '../../../layouts/PageBlock';
-import { Box, Grid } from '@mui/material';
+import { Box, Divider, Grid, Stack, Typography } from '@mui/material';
 import DetailDisplay from '../../../components/DetailDisplay';
 import WorkPackageStageChip from '../../../components/WorkPackageStageChip';
 import { timelinePipe } from '../../../utils/pipes';
+import { Construction, Work } from '@mui/icons-material';
+import ScheduleIcon from '@mui/icons-material/Schedule';
+import StackedLineChartIcon from '@mui/icons-material/StackedLineChart';
+import DoneAllIcon from '@mui/icons-material/DoneAll';
 
 interface WorkPackageDetailsProps {
   workPackage: WorkPackage;
 }
 
 const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage }) => {
   return (
-    <PageBlock
-      title={'Work Package Details'}
-      headerRight={
-        <Box>
-          {workPackage.stage ? <WorkPackageStageChip stage={workPackage.stage} /> : null}
-          <WbsStatus status={workPackage.status} />
-        </Box>
-      }
-    >
-      <Grid container spacing={1}>
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Lead" content={fullNamePipe(workPackage.projectLead)} paddingRight={2} />
+    <>
+      <Box sx={{ mb: 1 }}>
+        <Grid container spacing={2}>
+          <Grid item display="flex" alignItems="center" xs={6} sm={3}>
+            <Typography
+              variant="h5"
+              sx={{
+                mb: 1,
+                cursor: 'pointer'
+              }}
+            >
+              Work Package Details
+            </Typography>
+          </Grid>
+          <Grid item display="flex" alignItems="center" xs={6} sm={3}>
```

Comment:
i dont think this needs to be flex as its just one box.

Reply:
For whatever reason making it a flex keeps it aligned horizontally with the above typography